core/src/style/polygonStyle.cpp

```diff
@@ -65,56 +96,114 @@ void PolygonStyle::constructShaderProgram() {
     m_shaderProgram->setSourceStrings(fragShaderSrcStr, vertShaderSrcStr);
 }
 
-VboMesh* PolygonStyle::newMesh() const {
-    return new Mesh(m_vertexLayout, m_drawMode);
-}
+namespace {
```

Comment:
Ya, but that would mean that we have to pull in all includes and internal structs into the headers which I would like to avoid.